pkg/schedule/schedulers/balance_leader.go

```diff
@@ -86,10 +86,12 @@ func (conf *balanceLeaderSchedulerConfig) Update(data []byte) (int, any) {
 	newConfig, _ := json.Marshal(conf)
 	if !bytes.Equal(oldConfig, newConfig) {
 		if !conf.validateLocked() {
-			json.Unmarshal(oldConfig, conf)
+			_ = json.Unmarshal(oldConfig, conf)
```

Comment:
Should we handle the error?

Reply:
I can add a log for this. It seems like nothing else can be done.

Is it enough?

Reply:
witness has handled this error, you can refer to it.

Reply:
thanks for your suggestion. https://github.com/tikv/pd/pull/8346/commits/9de11548055277b10ec7dc329da86788eb114b95